integration-tests/bot/chainrpc.py

```diff
@@ -117,7 +117,7 @@ def list(self, name=DEFAULT_WALLET, type='staking', enckey=None):
         '''list addresses
         :param name: Name of the wallet. [default: Default]
         :params type: [staking|transfer]'''
-        return self.client.call('wallet_listStakingAddresses' if type == 'staking' else 'wallet_listTransferAddresses', [name, enckey or get_enckey()])
+        return self.client.call('wallet_listStakingAddresses' if type == 'staking' else 'wallet_listTransferAddresses', [name, enckey or get_enckey()],0,0,False)
```

Comment:
do these parameters now need to be provided? could in the API be put as `Option`, so if they are not provided, it'll just take a default value?

Reply:
agree, i'll try to use Option for default value